packages/@uppy/companion/src/server/controllers/s3.js

```diff
@@ -8,6 +9,13 @@ function rfc2047Encode (data) {
   return `=?UTF-8?B?${Buffer.from(data).toString('base64')}?=` // We encode non-ASCII strings
 }
 
+function isValidBucket (bucket) {
+  return typeof bucket === 'string' || bucket instanceof String
```

Comment:
hum no, and instance of `String` would not be correct here.
```suggestion
  return typeof bucket === 'string' && bucket !== ''
```